### PR TITLE
Update rubygem-foreman_expire_host to 8.2.0

### PR DESCRIPTION
**Pull request by ochnerd**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
-Version: 8.1.0
+Name: rubygem-%{gem_name}
+Version: 8.2.0
 Release: 1%{?foremandist}%{?dist}
 Summary: Foreman plugin for limiting host lifetime
-Group: Applications/Systems
 License: GPLv3
 URL: https://github.com/theforeman/foreman_expire_hosts
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-Source1: %{gem_name}.cron.d
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec:18]: Is this intentional? If so, I'd expect this file to be removed from the packaging repo too

**Review by ekohl — DISMISSED** [review]
Please use a commit message that says it's bumping the version to 8.2.0.

File: packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec (hunk 1 above)
Comment (ochnerd) [reply packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec:18]: Oh thanks for noticing, this was not intentional. I missed that the recreation of the spec from the template deleted that part.

Comment (ochnerd) [reply packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec:18]: I changed the commited and added `Source1` back into the spec.

**Comment by ochnerd** [comment]
This makes the automated MR obsolete https://github.com/theforeman/foreman-packaging/pull/10873

**Review by evgeni — APPROVED** [review]

**Pull request merged**